lib/zebra.h

```diff
@@ -346,6 +346,10 @@ typedef enum {
 	for (afi = AFI_IP; afi < AFI_MAX; afi++)                               \
 		for (safi = SAFI_UNICAST; safi < SAFI_MAX; safi++)
 
+#define FOREACH_AFI_SAFI_MAX(afi, safi, max)                                   \
```

Comment:
`max` seems to be always `SAFI_MPLS_VPN` in all references. Is this some sort of special value? Should we just hard-code it then instead of passing as a parameter everywhere?

Reply:
Just don't know how to name the macro in such a case. What do you suggest here? FOREACH_AFI_SAFI_SOMETHING(afi, safi)?

Reply:
Yeah, that's why I asked about `SAFI_MPLS_VPN` being a special value :)
Do you know why it is a maximum in all those cases? If we find an answer, we'll probably find a suitable name.

Reply:
I believe, it's only because of a graceful restart.

Reply:
How about `FOREACH_AFI_SAFI_NSF`?

Reply:
Makes sense :+1:

Reply:
Changed, thanks.